packages/sharing-smart-contract/contracts/Store.sol

```diff
@@ -69,27 +70,18 @@ abstract contract Store {
         uint48 duration; // 48 bit allows 89194 years of delay
     }
 
-    modifier onlyCollectionNotSubscribed(uint256 _collectionId) {
-        require(
-            lastSubscriptionExpiration[_collectionId] < block.timestamp,
-            "Collection has ongoing subscriptions"
-        );
-        _;
-    }
-
-    modifier onlyProtectedDataNotAvailableInSubscription(
-        uint256 _collectionId,
-        address _protectedData
-    ) {
-        require(
-            protectedDataInSubscription[_collectionId][_protectedData] == false,
-            "ProtectedData is available in subscription"
-        );
-        _;
-    }
     /***************************************************************************
-     *                       Renting                                      *
+     *                       Renting                                           *
      ***************************************************************************/
+    event ProtectedDataAddedToRenting(
```

Comment:
For renting is the correct term, in my opinion we would use the word 'to' for example, when adding the protected data to a collection 

```
The choice between "for renting" and "to renting" depends on the context and the intended meaning. However, it's important to note that "to renting" is not a standard phrase in English. The more common and grammatically correct expression is "for renting."

Here's a brief explanation:

For Renting:

This phrase is commonly used when describing something that is available or suitable for rent.
Example: "The apartment is for renting," or "The house is for renting."
In this context, "for" indicates the purpose or availability of the property for rent.
To Renting:

This construction is not typical in English, and it may be considered incorrect.
If you mean to convey the idea of renting something, it's more appropriate to use "for renting."
Example: Instead of saying "The car is to renting," you should say "The car is for renting.
```"

Reply:
in short, 
for   + verb
to   + noun